docs/overview/what_is__-__main_areas/con-monitoring-istio.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
this entire topic is not so much about "what is Istio", but how to configure it? So the file might be better suited somewhere else.